deploy/helm/charts/templates/csi-node.yaml

```diff
@@ -0,0 +1,127 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: {{ template "openebs-cstor.fullname" . }}-csi-node
+  {{- if .Values.csiNode.annotations }}
+  annotations:
+  {{ toYaml .Values.csiNode.annotations | indent 4 }}
+  {{- end }}
+  labels:
+    {{- include "openebs-cstor.csiNode.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "openebs-cstor.csiNode.matchLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "openebs-cstor.csiNode.labels" . | nindent 8 }}
+        {{- if .Values.csiNode.podLabels}}
+        {{ toYaml .Values.csiNode.podLabels | nindent 8 }}
+        {{- end}}
+    spec:
+      #priorityClassName: system-node-critical
```

Comment:
Are we setting this somewhere else?

Reply:
No... As it was commented in the source yaml. We can discuss on this can take into a next PR.

Reply:
@prateekpandey14 On the priority class, should we add it?

Reply:
added